### PR TITLE
HMPP support

### PR DESCRIPTION
**Pull request by minaEweida**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -87,8 +87,29 @@ void setupAllTargetsWithDefaultSourceSets(Project project) {
...
+                dependencies {
+
+                }
+            }
+
+            iosTest {
+                dependencies {
+
+                }
+            }
```
Comment (MarcoSignoretto) [inline build.gradle:104]: I think you can remove those

Comment (minaEweida) [reply build.gradle]: yeah, done

File: build.gradle [hunk 2]
```diff
@@ -87,8 +87,17 @@ void setupAllTargetsWithDefaultSourceSets(Project project) {
...
+        iosSimulatorArm64()
+
+        sourceSets {
+            iosSimulatorArm64Main {
+                dependsOn iosMain
+            }
+            iosSimulatorArm64Test {
+                dependsOn iosTest
+            }
+        }
```
Comment (carlonzo) [inline build.gradle:100]: can you make sure you really need this?

**Review by carlonzo — APPROVED** [review]
:+1:

File: build.gradle (hunk 2 above)
Comment (nikolaykasyanov) [reply build.gradle:100]: We do, that's exactly what our KMP DSL plugin does and what JetBrains recommend: https://kotlinlang.org/docs/mpp-share-on-platforms.html#use-target-shortcuts cc @minaEweida

**Review by nikolaykasyanov — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,7 @@ All notable changes to this project will be documented in this file.
 ---
 
 ## master
+* HMPP support
```
Comment (nikolaykasyanov) [inline CHANGELOG.md:7]: Should the `iosSimulatorArm64` slice be mentioned explicitly too?

Comment (minaEweida) [reply CHANGELOG.md:8]: Done

**Review by MarcoSignoretto — APPROVED** [review]

**Pull request merged**